### ISSUE TITLE
Can't load tokenizer for 'facebook/hubert-base-ls960'

### ISSUE THREAD

**Issue by harrypotter90** [huggingface/transformers#14214]
## Environment info

- `transformers` version: 4.12.2
- Platform: Mac
- Python version: 3.7
- PyTorch version (GPU?): 1.9
- Tensorflow version (GPU?): No
- Using GPU in script?: No
- Using distributed or parallel set-up in script?: No

I just run simple code to load Hubert pretrained base model 

```
from transformers import Wav2Vec2Processor, HubertForCTC
import torch
import librosa
PROCESSOR = Wav2Vec2Processor.from_pretrained('facebook/hubert-base-ls960')
model = HubertForCTC.from_pretrained('facebook/hubert-base-ls960')
```

And i got error trace:

```
Traceback (most recent call last):
  File "/Users/
    PROCESSOR = Wav2Vec2Processor.from_pretrained('facebook/hubert-base-ls960')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/transformers/models/wav2vec2/processing_wav2vec2.py", line 105, in from_pretrained
    tokenizer = Wav2Vec2CTCTokenizer.from_pretrained(pretrained_model_name_or_path, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/transformers/tokenization_utils_base.py", line 1733, in from_pretrained
    raise EnvironmentError(msg)
OSError: Can't load tokenizer for 'facebook/hubert-base-ls960'. Make sure that:

- 'facebook/hubert-base-ls960' is a correct model identifier listed on 'https://huggingface.co/models'
  (make sure 'facebook/hubert-base-ls960' is not a path to a local directory with something else, in that case)

- or 'facebook/hubert-base-ls960' is the correct path to a directory containing relevant tokenizer files
```

**Comment by patrickvonplaten** [comment]
Hey @harrypotter90,

Note that `facebook/hubert-base-ls960` is just the pretrained model and therefore does not have a tokenizer yet. You can create one yourself as shown in this blog post: https://huggingface.co/blog/fine-tune-wav2vec2-english

**Comment by kroq-gar78** [comment]
Thanks for the clarification @patrickvonplaten. The documentation is a bit misleading, since it has some references to loading a processor for `facebook/hubert-base-ls960`. (This also made me second-guess whether the model was fine-tuned at all.) Example: https://huggingface.co/transformers/model_doc/hubert.html#transformers.HubertForCTC

Can you update the documentation to use a different model that supports the tokenizer, such as `facebook/hubert-large-ls960-ft`?

**Comment by patrickvonplaten** [comment]
Hey @kroq-gar78,

Ah yeah, that's great feedback! I'll change that asap. The documentation should definitely use a model that has a tokenizer! 

Also I've updated the model card: https://huggingface.co/facebook/hubert-base-ls960 leaving a big note to make it clearer.

Thanks!

**Comment by dmatekenya** [comment]
I'm running through the same issue as above. I followed through this [blog-post](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2) but instead of saving the model to Hugging Face hub, I saved in my local directory. There is a checkpoint folder there with the preprocessor while the main directory has turkish-tokenizer_config.json file. However, when I load the model for evaluation, its still giving the tokenizer not found error. I would appreciate any help. Thanks.
